src/js/menu/acquia_lift.personalize.behaviors.js

```diff
@@ -240,7 +228,7 @@
                   switch (type) {
                     case 'option_sets':
                       // If the menu already has a link for this setting, abort.
-                      if (!$menu.find('[data-acquia-lift-personalize-agent="' + obj.agent + '"][data-acquia-lift-personalize-id="' + key + '"].acquia-lift-preview-page-variation').length) {
+                      if (!$menu.find('[data-acquia-lift-personalize-agent="' + obj.agent + '"][data-acquia-lift-personalize-id="' + key + '"].acquia-lift-preview-element-variation').length) {
```

Comment:
There is no equivalent usage now of acquia-lift-preview-element-variation.  All of the usages of the changed class were removed.  As a result - I think this class should just be removed rather than updated.